microsoft-365/compliance/customer-key-manage.md

```diff
@@ -336,7 +336,7 @@ Remove-AzKeyVaultAccessPolicy -VaultName Contoso-O365EX-NA-VaultA1 -UserPrincipa
 
 ## Roll back from Customer Key to Microsoft managed Keys
 
-If you need to revert to Microsoft-managed keys, you can. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys.
+If you need to revert to Microsoft-managed keys, you can in exchange. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys. Roll back is not supported in SPO. 
```

Comment:
```suggestion
If you need to revert to Microsoft-managed keys, you can do so in Exchange Online. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys. Roll back is not supported in SPO. 
```

Reply:
Slight change to Johan's suggestion.

In Exchange Online, if you need to revert ...
In SharePoint, you can't revert to Microsoft-managed keys, roll back is not supported.

Reply:
Thank you @KCCross. I've changed my suggestion accordingly.